hs_core/receivers.py

```diff
@@ -93,6 +97,8 @@ def metadata_element_pre_update_handler(sender, **kwargs):
         element_form = AbstractValidationForm(request.POST)
     elif element_name == "fundingagency":
         element_form = FundingAgencyValidationForm(request.POST)
+    elif element_name == "citation":
+        return {'is_valid': True, 'element_data_dict': {'value': request.POST.get('content')}}
```

Comment:
Same with a form here.

Reply:
I think the only rule is length, which is handled in multiple places.